# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/windows/sdn/sdnDocAccessible.h
# Commit: a53fb045a8e3
# Full Hash: a53fb045a8e34c1e1f14ca7b705eb9a5a0e76db2
# Author: James Teh <jteh@mozilla.com>
# Date: 2021-05-02 09:29:52
# Regressor Bug: 1694865
# File Overlap Count: 1
# Description:
#   Bug 1694865 part 19: Make EnumVariant, GeckoCustom and sdn*Accessible classes aggregate MsaaAccessible/MsaaDocAccessible instead of AccessibleWrap/DocAccessibleWrap. r=morgan
#   
#   These objects need to aggregate the primary COM object (MsaaAccessible).
#   Once the IUnknown implementation moves out of AccessibleWrap, it won't be possible to aggregate *AccessibleWrap any more.
#   
# ==============================================================================

diff -r 5bd374b642fe -r a53fb045a8e3 accessible/windows/sdn/sdnDocAccessible.h
--- a/accessible/windows/sdn/sdnDocAccessible.h	Sat May 01 21:09:04 2021 +0000
+++ b/accessible/windows/sdn/sdnDocAccessible.h	Sat May 01 22:29:25 2021 +0000
@@ -10,15 +10,14 @@
 #include "ISimpleDOM.h"
 #include "IUnknownImpl.h"
 
-#include "DocAccessibleWrap.h"
+#include "MsaaDocAccessible.h"
 
 namespace mozilla {
 namespace a11y {
 
 class sdnDocAccessible final : public ISimpleDOMDocument {
  public:
-  explicit sdnDocAccessible(DocAccessibleWrap* aAccessible)
-      : mAccessible(aAccessible){};
+  explicit sdnDocAccessible(MsaaDocAccessible* aMsaa) : mMsaa(aMsaa){};
   ~sdnDocAccessible(){};
 
   DECL_IUNKNOWN
@@ -44,7 +43,7 @@
       /* [in] */ BSTR __RPC_FAR* commaSeparatedMediaTypes);
 
  protected:
-  RefPtr<DocAccessibleWrap> mAccessible;
+  RefPtr<MsaaDocAccessible> mMsaa;
 };
 
 }  // namespace a11y